src/sys/windows/event.rs

```diff
@@ -14,25 +14,25 @@ pub fn token(event: &Event) -> Token {
 }
 
 pub fn is_readable(event: &Event) -> bool {
-    if is_hup(event) {
+    if is_error(event) {
         return true;
     }
-    (event.flags & (afd::KNOWN_AFD_EVENTS & !afd::AFD_POLL_SEND)) != 0
+    event.flags & (afd::AFD_POLL_RECEIVE | afd::AFD_POLL_ACCEPT | afd::AFD_POLL_DISCONNECT) != 0
```

Comment:
When does `AFD_POLL_DISCONNECT` trigger? Because libuv seems to think this match `EV_EOF`/`EPOLLRDHUP` (see https://github.com/libuv/libuv/commit/5ad6a9104e3a81934176bcda780945fae7eb9550) and we're using those in `is_read_hup`.

Reply:
I also have some more questions about the other flags, but I can't find any documentation on Windows' AFD.

Reply:
summoning @piscisaureus, the AFD expert.

Reply:
I think libuv is correct in this case, since libuv is battle-tested library.